architecture/additional_concepts/authentication.adoc

```diff
@@ -206,13 +206,136 @@ When using a service account as an OAuth client:
 $ oc sa get-token <serviceaccount name>
 ----
 
-* `redirect_uri` must match an annotation on the service account that has the
```

Comment:
A few lines up (currently lines 197 - 202) there are some <replaceables> that have a white space instead of underscore. Like <serviceaccount namespace>, which could be <serviceaccount_namespace> instead.